circleguard/utils.py

```diff
@@ -110,3 +110,27 @@ def span_to_list(span):
         else:
             ret.add(int(s))
     return ret
+
+def convert_ur(ur, mods, *, to):
+    """
+    Converts an unstable rate to a converted unstable rate, depending on the
+    mods the replay was played with.
+
+    Parameters
+    ----------
+    ur: float
+        The unconverted ur of the replay.
+    mods: Mod
+        The mods the replay was played with. Only ``Mod.DT`` and ``Mod.HT``
+        will affect the unstable rate conversion.
+    """
+    if to not in ["cv", "ucv"]:
+        raise ValueError(f"Expected one of cv, ucv. Got {to}")
+    conversion_factor = 1
+
+    if Mod.DT in mods:
+        conversion_factor = (1 / 1.5) if to == "cv" else 1.5
+    elif Mod.HT in mods:
+        conversion_factor = (1 / 0.75) if to == "cv" else 0.75
+
+    return ur * conversion_factor
```

Comment:
```suggestion
    if Mod.DT in mods:
        conversion_factor = 1.5
    elif Mod.HT in mods:
        conversion_factor = 0.75
    if to == "ucv":
        return ur * conversion_factor
    return ur / conversion_factor

```
This is way cleaner imo

Reply:
you got them backwards, but agree